config/examples/backend.yml

```diff
@@ -1,18 +1,25 @@
+base: &default
+  url: <%= ENV.fetch('BACKEND_URL', 'http://localhost:3001') %>
+  public_url: <%= ENV.fetch('BACKEND_PUBLIC_URL') { "https://backend-#{ENV['TENANT_NAME']}.#{ENV['THREESCALE_SUPERDOMAIN']}" } %>
+  persistent: <%= ENV.fetch('BACKEND_PERSISTENT', '1') == '1' %>
+  max_tries: <%= ENV.fetch('BACKEND_MAX_TRIES', 5) %>
```

Comment:
This defaults to 5 in the code anyway.

https://github.com/3scale/porta/blob/dc59523a892ae5fbe71cf446d40f9460033f462f/config/application.rb#L258